frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -57,7 +57,7 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   const preferredLanguage = personalInformation.preferredLanguageId ? await getLookupService().getPreferredLanguage(personalInformation.preferredLanguageId) : undefined;
   const maritalStatusList = await getLookupService().getAllMaritalStatuses();
   const dateString = personalInformation.birthDate ? new Date(personalInformation.birthDate).toLocaleDateString() : undefined;
-  const birthParsedFormat = dateString && isValidDate(dateString, 'mm/dd/yyyy', '/') ? toLocaleDateString(await parse(dateString, 'dd/mm/yyyy', new Date()), locale) : undefined;
+  const birthParsedFormat = dateString && isValid(parse(dateString, 'MM/dd/yyyy', new Date())) ? toLocaleDateString(await parse(dateString, 'MM/dd/yyyy', new Date()), locale) : undefined;
```

Comment:
Remove the `await` with `parse` since it's not an async func.

```suggestion
  const birthParsedFormat = dateString && isValid(parse(dateString, 'MM/dd/yyyy', new Date())) ? toLocaleDateString(parse(dateString, 'MM/dd/yyyy', new Date()), locale) : undefined;
```

Reply:
Fixed: e65505e3767c75f0249724a6be94068de16e778a

Reply:
Fixed: 5580940c8395943b8bc9313c68833a66bc3521da